plugins/volumeFilter_test.go

```
@@ -155,6 +155,28 @@ func TestMakeFilterVolume(t *testing.T) {
 	}
 }
 
+func TestMakeFilterVolume_FailsOnEmptyConfig(t *testing.T) {
+	testAssetDisplayFn := model.MakeSdexMappedAssetDisplayFn(map[model.Asset]hProtocol.Asset{model.Asset("XLM"): utils.NativeAsset})
+	tradingPair := &model.TradingPair{Base: "XLM", Quote: "XLM"}
+
+	configUnderTest := &VolumeFilterConfig{}
+	_, e := makeFilterVolume(
+		"someConfigValue",
+		"someExchangeName",
+		tradingPair,
+		testAssetDisplayFn,
+		utils.NativeAsset,
+		utils.NativeAsset,
+		&sql.DB{},
+		configUnderTest,
+	)
+	if !assert.Error(t, e) {
```

new test with an empty config should return an error as name of test suggests